requirements/base.txt

```diff
@@ -2,7 +2,7 @@
 
 # Note: also adjust pootle/constants::DJANGO_MINIMUM_REQUIRED_VERSION
 # and docs/releases/M.N.O.rst
-Django~=1.10.8  # rq.filter: <1.11
+Django~=1.11.12  # rq.filter: <1.13
```

Comment:
@phlax This would probably install latest Django 1.12.x where the intention seems to be to make it 1.11.x